### DISCUSSION TITLE
Restore Filters for NFT Collections on Mobile App

### DISCUSSION THREAD

**Discussion by P-r-a-s-a-n-n-a** [spacescan-io/Spacescan-discussion#74] [Support]
![Screenshot_20230804-200150_Discord](https://github.com/spacescan-io/Spacescan-discussion/assets/89355713/2253d8bd-a3a5-4839-8db6-be2b0ab7d84c)

**Comment by lakshmirvm** [comment]
Filter restored in mobile design.
